entity-framework/core/extensions/index.md

```diff
@@ -449,3 +455,12 @@ NCache Entity Framework Core Provider is a distributed second level cache provid
 Life cycle hooks (for SaveChanges). For EF Core: 2-3.
 
 [GitHub repository](https://github.com/JValck/Ramses) | [NuGet](https://www.nuget.org/packages/Ramses)
+
+
+### EntityFrameworkCore.Extensions
+
+An extension library for Dynamic Data Masking (SQL Server) and MigrationBuilder and ModelBuilder extensions. For EF Core: 5.
+
+An updated fork for the data masking feature can be found at [EntityFrameworkCore.Extensions.SqlServer.DataMasking](https://github.com/sander1095/EntityFrameworkCore.Extensions.SqlServer.DataMasking)
```

Comment:
Not needed

Reply:
Are you sure? What if people get linked to this item, or find this item first on the page.

In that case they won't know there's an updated version available, which can only cause confusion.